Userland/Libraries/LibCore/MimeData.cpp

```diff
@@ -79,4 +79,31 @@ String guess_mime_type_based_on_filename(const StringView& path)
     return "text/plain";
 }
 
+#define ENUMERATE_HEADER_CONTENTS(V)                                                                   \
+    V(pbm, "image/x-portable-bitmap", 3, 0x50, 0x31, 0x0A)                                             \
+    V(pgm, "image/x-portable-graymap", 3, 0x50, 0x32, 0x0A)                                            \
+    V(png, "image/png", 8, 0x89, 'P', 'N', 'G', 0x0D, 0x0A, 0x1A, 0x0A)                                \
+    V(ppm, "image/x-portable-pixmap", 3, 0x50, 0x33, 0x0A)                                             \
+    V(gif_87, "image/gif", 6, 'G', 'I', 'F', '8', '7', 'a')                                            \
+    V(gif_89, "image/gif", 6, 'G', 'I', 'F', '8', '9', 'a')                                            \
+    V(bmp, "image/bmp", 2, 'B', 'M')                                                                   \
+    V(jpeg, "image/jpeg", 4, 0xFF, 0xD8, 0xFF, 0xDB)                                                   \
+    V(jpeg_jfif, "image/jpeg", 12, 0xFF, 0xD8, 0xFF, 0xE0, 0x00, 0x10, 'J', 'F', 'I', 'F', 0x00, 0x01) \
+    V(jpeg_huh, "image/jpeg", 4, 0xFF, 0xD8, 0xFF, 0xEE)                                               \
+    V(shell, "text/x-shellscript", 10, '#', '!', '/', 'b', 'i', 'n', '/', 's', 'h', '\n')
```

Comment:
I wonder if it would be useful to alphabetically sort this list? Could get messy once more are added :)